ios/RNAInputAudioStream.h

```diff
@@ -3,7 +3,6 @@
 #import "RNAudioException.h"
 
 enum AUDIO_FORMATS {
-  PCM_8BIT = 1,
```

Comment:
This is an uncommon format and not something nicely supported and packaged as an [AVAudioCommonFormat](https://developer.apple.com/documentation/avfaudio/avaudiocommonformat?language=objc) so I felt like getting rid of this was fine. If there's a big ask for it it's not too difficult to support